Automattic/woocommerce-payments
# Include discount in fees brakedown tooltip

Fixes #7750 

#### Changes proposed in this Pull Request

Applies the discount rate to all fees in the tooltip so that the values match with actual fees. 

#### Testing instructions

1. Add any custom fee to any payment method
2. Go to `payments` -> `settings` 
3. The fees pill should have `From ...` 
4. That values should match the base fee in the tooltip
-------------------


**Post merge**
